### PR TITLE
Image components and responsivity

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Another part for #10 

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

I have used https://css-tricks.com/a-guide-to-the-responsive-images-syntax-in-html/#being-more-chill-about-sizes as a start.
It does not recalculate the exact width of the image, just something that would be close to that, and the `sm` and lower size seems to load all device width and does not take the `devicePixelRatio` into the account.

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by Simply007** [comment]
I am still not sure why mobile devices that have `sizes` property set to 100vw is 1080 on i.e. pixel 3a, because viewport should be 394 - https://yesviz.com/devices/google-pixel-3a/.

https://github.com/Kentico/kontent-starter-corporate-next-js/pull/22/files#diff-3b5149490082e66ab828458e4c862f3235004dd6460ca6df4b3a1bf839519fe2R32

File: components/sections/ContentSection.js [hunk 1]
```diff
@@ -31,7 +33,10 @@ function ContentSection(props) {
           {get(section, 'image.value[0]', null) && (
             <Grid item xs={12} sm={6} className={classes.column}>
               {/* TODO use Next Image Component */}
```
Comment (makma) [inline components/sections/ContentSection.js:35]: remove

**Review by makma — APPROVED** [review]
LGTM :)

File: components/footerSections/ContentSection.js [hunk 2]
```diff
@@ -21,7 +22,13 @@ function ContentSection(props) {
       {get(section, 'image.value[0]', null) && (
         <div>
           {/* TODO use Next Image Component */}
```
[the remark this answers is not in the document]
Comment (Simply007) [reply components/footerSections/ContentSection.js:24]:
```suggestion

```

File: components/sections/ContentSection.js (hunk 1 above)
Comment (Simply007) [reply components/sections/ContentSection.js:35]:
```suggestion

```

**Pull request merged**